ERCS/erc-xxx.md

```diff
@@ -0,0 +1,441 @@
+---
```

Comment:
```suggestion
---
eip: 7730
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.